python/jupyterlab-chat/jupyterlab_chat/ychat.py

```diff
@@ -52,105 +54,105 @@ def yusers(self) -> Map:
     def ymetadata(self) -> Map:
```

Comment:
Does `pycrdt` have generic types, e.g. allowing us to type `ymetadata` as `Map[str, Any]`? 

This would be more helpful for the `ymessages` and `yusers` properties, which are currently typed as just `Array` and `Map` respectively.

Reply:
Not that I know. But these methods should only be uses to add observers, not to use the data.

Maybe @davidbrochart has a better idea on it.